source/types/types.ts

```diff
@@ -210,6 +210,12 @@ export enum Unit {
 	LONGITUDE = "degrees_east",
 }
 
+export enum Provider {
```

Comment:
Perhaps I'm being pedantic, but I have concerns with the naming:
- `Provider` is very general, so there could be namespace issues in other people's code. For example, React has its own concept of providers.
- `Provider` works when used like `Provider.WOD`, however `Object.values(Provider)` or `for (provider in Provider)` looks kind of odd. I'd naively expect Provider to be a class because that's a common naming practice in the JS community.
- We prefix our interfaces, but not other types. Would it make sense to start postfixing enums like `ProviderEnum` for clarity?

Reply:
Lets use Enum postfix. That will make it a lot clearer